etc/mpv.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.mpv.Mpv</id>
+    <name>mpv</name>
+    <summary>A free, open source, and cross-platform media player</summary>
+    <description>
+        <p>Mpv is a free (as in freedom) media player for the command line. It supports a wide variety of media file formats, audio and video codecs, and subtitle types.</p>
+        <p>Mpv has an OpenGL, Vulkan, and D3D11 based video output that is capable of many features loved by videophiles, such as video scaling with popular high quality algorithms, color management, frame timing, interpolation, HDR, and more.</p>
+        <p>While mpv strives for minimalism and provides no real GUI, it has a small controller on top of the video for basic control.</p>
+        <p>Mpv can leverage most hardware decoding APIs on all platforms. Hardware decoding can be enabled at runtime on demand.</p>
+        <p>Powerful scripting capabilities can make the player do almost anything. There is a large selection of user scripts on the wiki.</p>
+        <p>A straightforward C API was designed from the ground up to make mpv usable as a library and facilitate easy integration into other applications.</p>
+    </description>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-2.0+ AND LGPL-2.0-or-later</project_license>
+    <developer_name>mpv.io</developer_name>
+    <launchable type="desktop-id">mpv.desktop</launchable>
+    <translation type="gettext">mpv</translation>
```

Comment:
this is incorrect, mpv is not translatable

Reply:
Good catch. Fixed.